oem/ibm/test/libpldmresponder_oem_platform_test.cpp

```diff
@@ -206,34 +206,6 @@ TEST(generateStateEffecterOEMPDR, testGoodRequest)
     auto event = sdeventplus::Event::get_default();
     std::unique_ptr<CodeUpdate> mockCodeUpdate =
         std::make_unique<MockCodeUpdate>(mockDbusHandler.get());
-    GetSubTreeResponse mockResponse1 = {
-        {"/xyz/openbmc_project/inventory/system/chassis/motherboard/dcm0/cpu0",
-         {{"xyz.openbmc_project.Inventory.Manager",
-           {"xyz.openbmc_project.Inventory.Item.Cpu"}}}}};
-
-    EXPECT_CALL(*mockDbusHandler,
-                getSubtree("/xyz/openbmc_project/inventory/system", 0,
-                           std::vector<std::string>{
-                               "xyz.openbmc_project.Inventory.Item.Cpu"}))
```

Comment:
Similar to the EXPECT_CALL of getsubtree, we should call EXPECT_CALL for getsubtreepaths.

Reply:
I don't think so, the test cases are working fine.